Extension/package.nls.json

```diff
@@ -843,6 +845,7 @@
         ]
     },
     "c_cpp.configuration.debugShortcut.description": "Show the \"Run and Debug\" play button and \"Add Debug Configuration\" gear in the editor title bar for C++ files.",
+    "c_cpp.configuration.persistVsDeveloperEnvironment.markdownDescription": "Remember the last used Visual Studio developer environment for the current workspace. This setting is only applicable for Windows.",
```

Comment:
`markdownDescription` should be `description` because there is no markdown in the text.

Reply:
fixed